### PR TITLE
Support multiple files as input

### PR DESCRIPTION
**Pull request by pitlv2109**
#12394

### CODE REVIEW

**Comment by pitlv2109** [comment]
This PR is ready for review. Low risk. PTAL @yangminzhu @esnible

File: istioctl/cmd/auth.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 	}
 
 	upgradeCmd = &cobra.Command{
-		Use:   "upgrade -f <yaml-file>",
```
Comment (yangminzhu) [inline istioctl/cmd/auth.go]: Don't we need the `-f` anymore?

Comment (pitlv2109) [reply istioctl/cmd/auth.go]: I think `Use` ignore hardcoded flags? `TestAuthUpgrade` from `auth_test.go` still passes. Plus, many places do not put flags in `Use`.
https://github.com/istio/istio/blob/master/istioctl/cmd/convert_ingress.go#L185

Comment (yangminzhu) [reply istioctl/cmd/auth.go]: I mean what's the point of removing `-f` but leaving `<yaml-file>` there?

Comment (yangminzhu) [reply istioctl/cmd/auth.go]: Also it's better to update the `Example` to call out it supports multiple files.

Comment (pitlv2109) [reply istioctl/cmd/auth.go]: Updated for both. PTAL.

File: istioctl/cmd/auth.go [hunk 2]
```diff
@@ -100,12 +100,12 @@ Kubernetes cluster or from a yaml file specified from command line.
 THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 `,
 		Example: `  # Upgrade the Istio authorization policy with service definition from the current k8s cluster:
-  istioctl experimental auth upgrade -f istio-authz-v1-policy.yaml
+  istioctl experimental auth upgrade -f istio-authz-v1-policy-1.yaml istio-authz-v1-policy-2.yaml
```
Comment (yangminzhu) [inline istioctl/cmd/auth.go]: Should it be `,` separated string? Not sure if whitespace works here.

Comment (pitlv2109) [reply istioctl/cmd/auth.go]: You're right. It won't work. Edited.

**Pull request merged**